### ISSUE TITLE
Decide between minPrice/TickWillingToTrade instead of Min/MaxTick 

### ISSUE THREAD

**Issue by eljhfx** [duality-labs/duality#151]
Good feature for arbitrageurs who can specify what price they want to stop trading at (which would be the "no arbitrage" price)

**Comment by teddyknox** [comment]
Related to #142

**Comment by teddyknox** [comment]
I would argue we should allow limit trading behind enemy lines as a much more ergonomic way of formulating this task.

**Comment by eljhfx** [comment]
I find that's much less useful to do especially since it's needed functionality for limit orders anyways.

Arbitrageurs want the bare minimum functionality for executing an atomic arbitrage i.e., trading up to some price.

I also think having this feature can be useful for all swaps not just limit orders, in preventing overly large, unreasonable transactions (it's basically a more effective MaxPrice in my opinion)

tldr, it is useful for limit orders, but also useful for arbitrage and preventing some weird DoS edge cases on swaps.
